coop_default_pricetag/demo/product_print_category.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record model="product.print.category" id="product_print_category.demo_category">
+        <field name="name">Default Category</field>
+        <field name="field_ids" eval="[(4, ref('product.field_product_product__name')),(4, ref('product.field_product_product__list_price'))]" />
+        <field name="pricetag_model_id" ref="coop_default_pricetag.pricetag_model_default"/>
+    </record>
+
```

Comment:
I don't undertstand. Why are you moving this to demo ?

Please don't. keep it where it was